pkg/controller/cleanup/cleanup.go

```diff
@@ -105,12 +105,12 @@ func (c *Controller) HandleCleanup() http.Handler {
 			}
 		}()
 
-		// Verification codes - purge codes from database entirerly.
+		// Verification codes - purge codes from database entirely.
 		// Their code/long_code hmac values will have been set to "".
 		func() {
 			defer observability.RecordLatency(ctx, time.Now(), mLatencyMs, &result, &item)
 			item = tag.Upsert(itemTagKey, "VERIFICATION_CODE")
-			if count, err := c.db.PurgeVerificationCodes(c.config.VerificationCodeMaxAge); err != nil {
```

Comment:
Currently this will purge at 48 hours. This will never be 'recycled' at 336 hours, because it's already gone.